WebHostLib/misc.py

```diff
@@ -37,17 +37,31 @@ def start_playing():
     return render_template(f"startPlaying.html")
 
 
-@app.route('/weighted-settings')
+# TODO for back compat. remove around 0.4.5
+@app.route("/weighted-settings")
 @cache.cached()
-def weighted_settings():
-    return render_template(f"weighted-settings.html")
+def route_weighted_options():
```

Comment:
```suggestion
def weighted_settings():
```
also `@cache.cached()` should go, sadly i can't make a suggestion for this